parm/atm/berror/staticb_identity.yaml

```diff
@@ -0,0 +1,2 @@
+covariance model: FV3JEDI-ID
```

Comment:
Just curious.   Is `FV3JEDI-ID` a standard JEDI variable or name for the identity model?   I can't find `FV3JEDI-ID` defined in `feature/single_anl`

Reply:
This is the FV3-JEDI YAML entry for the identity **B**, see for example: https://github.com/JCSDA-internal/fv3-jedi/blob/7e9e61222e32e494482dd442d6e86d9d31098c81/test/testinput/hyb-3dvar_geos.yaml#L29

Reply:
Thanks, @CoryMartin-NOAA .   There's still much for me to learn.